src/commands.rs

```diff
@@ -320,6 +320,10 @@ pub struct ElectrumOpts {
         default_value = "10"
     )]
     pub stop_gap: usize,
+
+    /// Enable or disable domain validation when connecting to Electrum servers.
+    #[clap(name = "VALIDATE_DOMAIN", long = "validate_domain", takes_value(false))]
```

Comment:
you shouldn't need `takes_value(false)`
```suggestion
    #[clap(name = "VALIDATE_DOMAIN", long = "validate_domain")]
```

Reply:
sure!